<launch>
  <!-- 加载话题配置 -->
  <rosparam file="$(find task_mgr)/config/topics.yaml" command="load" />
  
  <node pkg="task_mgr" type="task_mgr_node.py" name="task_mgr_node" output="screen">
  </node>
</launch>